changelog.txt

```diff
@@ -1,3 +1,11 @@
+4.11.0
+============
+Feature Requests: **MSAL.NET will now remove accounts from the cache that have expired refresh tokens** MSAL.NET will remove the refresh token and the associated account if the if suberror is "bad_token" to avoid overloading ESTS. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1720)
+
+Bug Fixes:
+**CryptographicException: Bad PKCS7 padding in iOS brokered authentication** ? [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1628)
```

Comment:
@jennyf19 not sure how to explain what is happening here based on whats in the issue, can you elaborate? also, is this the right issue for th iOS fix?

Reply:
Xamarin.iOS Applications using broker, migrated from ADAL.NET to MSAL.NET used to crash. This is now fixed.

Reply:
I would say this:
**When migrating a Xamarin application from ADAL.NET to MSAL.NET and preserving the keychain, a CryptographicException can be thrown from the BrokerKeyHelper.** MSAL.NET now does the broker key keychain look up by Service and Account only. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1628)

---
In reply to: [403313292](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1741#discussion_r403313292) [](ancestors = 403313292)

Reply:
@jmprieur - is this ok?

---
In reply to: [403326603](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1741#discussion_r403326603) [](ancestors = 403326603,403313292)